impargo/react-here-maps
# feat: use here maps js npm package

BREAKING CHANGE: this also changes the engine when using vector tiles from WEBGL to HARP. Additionally, if a client relied on @types/heremaps, they will encounter some type incompatibilities and might needs to adapt a few things. Some other props were also removed because they no longer make since now that the SDK is bundled, like `onScriptLoadError` and `secure`.

## What the PR Includes
- [x] Use HERE Maps JS bundles as an npm package instead of loading the scripts via loadjs.
- [x] Made a few improvements to types and removed multiple @ts-ignores.

Closes POS-2016

## Checklist
- [ ] Tests are added.
- [ ] Manual testing instructions are added.
- [ ] Configs are added/adapted if applicable.
- [x] Issue is properly linked if applicable.

## Manual Testing Instructions
1. Needs to be tested extensively with the CargoApps. For reference check PR https://github.com/impargo/impargo-apps/pull/9572
